app/services/custom_derivative_service.rb

```diff
@@ -3,7 +3,7 @@
 # @api public
 module CustomDerivativeService
   def valid?
-    true if file_set.rdf_type&.join&.downcase&.include?("intermediate_file")
+    file_set.rdf_type&.join&.downcase&.include?(Hyrax::ConditionalDerivativeDecorator::INTERMEDIATE_FILE_TYPE_TEXT)
```

Comment:
for ref: https://github.com/scientist-softserv/utk-hyku/blob/7cb55875f6f35eb70ec9354a3a93d78950197273/lib/extensions/hyrax/conditional_derivative_decorator.rb#L5